pkgs/applications/graphics/ImageMagick/default.nix

```diff
@@ -11,7 +11,7 @@ let
     else throw "ImageMagick is not supported on this platform.";
 
   cfg = {
-    version = "6.9.3-9";
+    version = "6.9.5-2";
     sha256 = "0q19jgn1iv7zqrw8ibxp4z57iihrc9kyb09k2wnspcacs6vrvinf";
```

Comment:
@fpletz Oh I wish you haven't done that :/

I deliberately left out the the mingw port's patch-set since it needs Windows testing...

You can find the latest patches here: https://github.com/Alexpux/MINGW-packages/tree/master/mingw-w64-imagemagick

But they're for 7 branch exclusively... The last branch 6 release is in https://github.com/Alexpux/MINGW-packages/tree/267230e3524678a0c41161d0c8de215ffce73a7c/mingw-w64-imagemagick if you're going to mess around with that.  But since I don't have windows...